coordinator/statement_executor.go

```diff
@@ -1312,6 +1327,8 @@ func (e *StatementExecutor) NormalizeStatement(stmt influxql.Statement, defaultD
 			case *influxql.DropSeriesStatement, *influxql.DeleteSeriesStatement:
 				// DB and RP not supported by these statements so don't rewrite into invalid
 				// statements
+			case *influxql.ShowTagValuesStatement:
+				// SHOW TAG VALUES should span multiple RPs if one is not specified.
```

Comment:
We are losing the default database insertion here, that seems bad

Reply:
We set the default database at the statement level, but not per measurement:
https://github.com/influxdata/influxdb/blob/0f79350cf06854692608dcfdb24942b52b2a69f6/coordinator/statement_executor.go#L1313

Reply:
Is there a test that shows this somehow? (default database and all rp's, and also
 default database and non-default rp)

Reply:
Default database, non-default RP: https://github.com/influxdata/influxdb/blob/0f79350cf06854692608dcfdb24942b52b2a69f6/tests/server_test.go#L8355

Default database, default RP: https://github.com/influxdata/influxdb/blob/0f79350cf06854692608dcfdb24942b52b2a69f6/tests/server_test.go#L8314